docs/lib/geo/fragments/js/search.md

```diff
@@ -0,0 +1,209 @@
+<amplify-callout>
+
+**Note:** Amplify Geo is in developer preview and is not intended to be used in production environments. Please reach out to us for any feedback and/or issues [here](https://github.com/aws-amplify/amplify-js/issues)
+
+</amplify-callout>
+
+> Prerequisite: [Install and configure](~/cli/start/install.md) the Amplify CLI
+
+## Provision a Search resource
+
+The primary way to provision Geo resources is through the Amplify CLI. Currently, you need to install the CLI with the `@geo` tag in order to get the Geo functionality. You can use the following command to install this version globally.
+
+```bash
+npm i -g aws-amplify@geo
+```
+
+Once that is complete, you can run the following command from your project's root folder to add a `geo` resource:
+
+```bash
+amplify add geo
+```
+
+```
+? Select which capability you want to add: (Use arrow keys)
+  Map (visualize the geospatial data)
+❯ Location search (search by places, addresses, coordinates)
+```
+
+From here you can follow the prompts to generate your new Place Index to be used for Search.
+
+<!-- TODO: replace with proper link to CLI docs -->
+For more information, you can visit the full [Amplify CLI Geo Search docs](~/lib/geo/search.md).
+
+## Set up your app with the AWS Amplify Geo category
+
+First, make sure you've provisioned a search resource by running `amplify add geo`, selecting the **Location search (search by places, addresses, coordinates)** option and running `amplify push` to deploy
+
+Next ensure to have the [Amplify configuration step](~/lib/geo/getting-started.md) done in your app's root entry point.
+
+Then, bring in the `Geo` category package where you need it:
+
+```javascript
+import { Geo } from 'aws-amplify';
+```
+
+## Add location search functionality on map
+
+You can use open source plugin [maplibre-gl-geocoder](https://github.com/maplibre/maplibre-gl-geocoder) with Amplify Geo to add fully functional search component on your map. Install maplibre-gl-geocoder with the following command:
+
+```bash
+npm install -S @maplibre/maplibre-gl-geocoder
+```
+
+If you haven't already add maplibre-gl-js and maplibre-gl-js-amplify to your app with `yarn` or `npm`:
+
+```bash
+npm install -S maplibre-gl maplibre-gl-js-amplify
+```
```

Comment:
```suggestion
```bash
npm install -S @maplibre/maplibre-gl-geocoder maplibre-gl maplibre-gl-js-amplify
```
```

Reply:
this doesn't flow with the previous line "Install `maplibre-gl-geocoder` with the following command:"